react/features/base/config/middleware.web.ts

```diff
@@ -0,0 +1,48 @@
+import { setUserFilmstripWidth } from '../../filmstrip/actions.web';
+import MiddlewareRegistry from '../redux/MiddlewareRegistry';
+import { updateSettings } from '../settings/actions';
+
+import { SET_CONFIG } from './actionTypes';
+import './middleware.any';
+
+/**
+ * The middleware of the feature {@code base/config}.
+ *
+ * @param {Store} store - The redux store.
+ * @private
+ * @returns {Function}
+ */
+MiddlewareRegistry.register(store => next => action => {
+    const result = next(action);
+
+    switch (action.type) {
+    case SET_CONFIG: {
```

Comment:
This block is repeated in both middlewares. What are you trying to do here? This PR is large already, please undo these unnecessary changes to the config middleware, they have nothing to do with PiP, AFAICT.